appengine/search/google-checks.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0"?>
```

Comment:
Oh I can just add them to the files in this repo! I didn't know that. I will do that in the next PR